### DISCUSSION TITLE
精心布局的开源 - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#95] [Announcements]
# 精心布局的开源 - 「开源之道」

https://opensourceway.community/posts/opensource/open_by_design/

**Comment by ZoomQuiet** [comment]
> 是也乎,(￣▽￣)

终于找到为什么对 开源 总是感觉哪里有不对的原因了,
相比 自由软件, 完全是路见不平一声吼式的自由弥漫...

开源软件从概念的创立到组织的经营, 无论哪个层面都充满了大教堂一般的精确严密,
完全就是一群商业精英在理解了自由软件后, 针对性的设计了一套全新管理方案,
虽然各种宣传, 其实缺少了最核心的一种自由后,
开源可以说就是面向商业的经营战略战术了...

相比自由软件, 更加专注恰饭...两边讨好...

所以, 感觉, 一定将在什么历史阶段爆雷...?
